src/modules/graphics/opengl.c

```diff
@@ -820,16 +820,7 @@ static void lovrGpuBindPipeline(Pipeline* pipeline) {
         glEnable(GL_STENCIL_TEST);
       }
 
-      GLenum glMode = GL_ALWAYS;
-      switch (state.stencilMode) {
-        case COMPARE_EQUAL: glMode = GL_EQUAL; break;
-        case COMPARE_NEQUAL: glMode = GL_NOTEQUAL; break;
-        case COMPARE_LESS: glMode = GL_GREATER; break;
-        case COMPARE_LEQUAL: glMode = GL_GEQUAL; break;
-        case COMPARE_GREATER: glMode = GL_LESS; break;
-        case COMPARE_GEQUAL: glMode = GL_LEQUAL; break;
-        default: break;
-      }
+      GLenum glMode = convertCompareMode(state.stencilMode);
```

Comment:
Hey so I'm not sure if `convertCompareMode` will work here because it does the sensible thing where e.g. `COMPARE_LESS` maps to `GL_LESS`.

But notice how this switch statement here is doing more of a "reverse compare mode" thing where `COMPARE_LESS` maps to `GL_GREATER`.  This is because it's more intuitive to think about it this way in the LÖVR API:

- In LÖVR `lovr.graphics.setStencilTest('less', 3)` means "only render pixels where stencil buffer is less than 3".
- In OpenGL `glStencilFunc(GL_LESS, 3, 0xff)` means "only render pixels where 3 < stencil buffer".

So we reverse it.

I think I'm going to need to do this reversed-compare-mode thing in another place soon in #143 so feel free to create a `convertCompareModeReversed` if you'd like, otherwise it's fine to just revert this change.